04_verify.sh

```diff
@@ -297,6 +297,22 @@ for container in ${EXPTD_CONTAINERS}; do
   iterate check_container "$container"
 done
 
+AUTHENTICATED_ENDPOINTS="${IRONIC_URL}nodes ${IRONIC_URL}nodes/detail"
+for endpoint in ${AUTHENTICATED_ENDPOINTS}; do
+    status="$(curl -sk -o /dev/null -I -w "%{http_code}" "$endpoint")"
+    if [ "$status" == 401 ]; then
+        echo "OK - Ironic $endpoint endpoint is secured"
+    elif [[ "$status" == 200 || "$status" == 405 ]]; then
```

Comment:
405 is not really possible for these two

Reply:
although [doc](https://docs.openstack.org/api-ref/baremetal/?expanded=agent-lookup-detail,agent-heartbeat-detail#agent-heartbeat) says return code could be  200, 400, 404, I'm getting 405
```
[airshipci@fmuyassarov metal3-dev-env]$ curl -k  https://172.22.0.2:6385/v1/heartbeat/
{"error_message": "{\"code\": 405, \"title\": \"Method Not Allowed\", \"description\": \"\"}"}[airshipci@fmuyassarov metal3-dev-env]$ 

```